docs/h5py_dataset.rst

```diff
@@ -231,6 +248,16 @@ picked them up for us:
 >>> print(train_set.provides_sources) # doctest: +SKIP
 [u'image_features', u'targets', u'vector_features']
 
+It also parsed axis labels, which are accessible through the ``axis_labels``
+property, which is a dict mapping source names to a tuple of axis labels:
+
+>>> print(train_set.axis_labels['image_features']) # doctest: +SKIP
+(u'batch', u'channel', u'height', u'width')
```

Comment:
I'm skipping these lines in the doctests (because of the Python2/3 difference in how unicode strings are displayed), but sure, I can remove the `u`s for consistency.